languages/zh-TW.js

```diff
@@ -0,0 +1,188 @@
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} 的賦予值必須為正好 ${min}${suffix}。`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} 的賦予值必須落在 ${min} 和 ${max}${suffix}之間。`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} 的賦予值必須大於 ${min}${suffix}。`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} 的賦予值必須小於 ${max}${suffix}。`,
+			REACTIONHANDLER_PROMPT: '你想穿梭到哪個頁面？',
+			COMMANDMESSAGE_MISSING: '輸入結尾缺少了一個以上的必要參數。',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} 是必要參數。`,
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `缺少了一個必選參數：(${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `你的選項不符合要求：(${possibles})`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time, abortOptions) => `${tag} | **${error}** | 你有 **${time}** 秒的時間，對提示訊息回覆一有效參數。輸入 **${abortOptions.join('**, **')}** 以便中止提示訊息。`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPEATING_REPROMPT: (tag, name, time, cancelOptions) => `${tag} | **${name}** 是一個重複的參數 | 你有 **${time}** 秒的時間， 對提示訊息回覆其他的有效參數。輸入 **${cancelOptions.join('**, **')}** 以便結束提示訊息。`,
+			MONITOR_COMMAND_HANDLER_ABORTED: '中止',
+			MONITOR_COMMAND_HANDLER_POSSIBILITIES: ['中止', '停止'],
+			MONITOR_COMMAND_HANDLER_REPEATING_POSSIBLITIES: ['取消'],
+			INHIBITOR_COOLDOWN: (remaining) => `你剛剛才用過這個指令。請等 ${remaining} 秒後再試一次。`,
+			INHIBITOR_DISABLED_GUILD: '此指令被這個伺服器的管理員停用了。',
+			INHIBITOR_DISABLED_GLOBAL: '此指令被 BOT 持有者停用了。',
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `權限不足，缺少：**${missing}**`,
+			INHIBITOR_NSFW: '你只能在 NSFW 頻道使用 NSFW 指令。',
+			INHIBITOR_PERMISSIONS: '你沒有權限使用這個指令。',
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `此伺服器缺漏 **${settings.join(', ')}** ${settings.length !== 1 ? '等' : ''}的伺服器設定，因此無法執行該指令。`,
+			INHIBITOR_RUNIN: (types) => `此指令只能在 ${types} 的頻道中使用。`,
+			INHIBITOR_RUNIN_NONE: (name) => `指令 ${name} 並沒有被設定為在任何頻道中執行。`,
+			COMMAND_BLACKLIST_DESCRIPTION: '讓機器人對使用者或伺服器封鎖或取消封鎖。',
+			COMMAND_BLACKLIST_SUCCESS: (usersAdded, usersRemoved, guildsAdded, guildsRemoved) => [
+				usersAdded.length ? `**名單已追加使用者**\n${util.codeBlock('', usersAdded.join(', '))}` : '',
+				usersRemoved.length ? `**名單已移除只用者**\n${util.codeBlock('', usersRemoved.join(', '))}` : '',
+				guildsAdded.length ? `**名單已加入伺服器**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
+				guildsRemoved.length ? `**名單已移除伺服器**\n${util.codeBlock('', guildsRemoved.join(', '))}` : ''
+			].filter(val => val !== '').join('\n'),
+			COMMAND_EVAL_DESCRIPTION: '直接運行 Javascript。僅限機器人持有者使用。',
+			COMMAND_EVAL_EXTENDEDHELP: [
+				'eval 指令直接運行程式碼，拋出的任何錯誤都會被處理。',
+				'此外還提供旗標以調整一些細項。輸入 --silent, --depth=number 或 --async 來調整輸出格式。',
+				'輸入 --silent 旗標，可以隱藏輸出結果。',
+				'輸入 --depth 旗標並賦予一個數字，可以調整 util.inspect 的 depth。',
+				'輸入 --async 旗標，可以將程式碼包在一個 async function，然後就能隨心所欲使用 await；若你想要有回傳值，就要有關鍵詞 return 。',
+				'輸入 --showHidden 旗標，可以啟用 util.inspect 的 showHidden 項目。',
+				'若輸出結果太長，將會包成一個檔案；機器人沒有 ATTACH_FILES 權限的話，則會直接列在 console 中。'
```

Comment:
I have yet to ask @bdistin about reducing the technicality of the language strings, `ATTACH_FILES` is a developer thing, users only know as "Attach Files" from the permission manager in Discord. Me personally, I have been using the names from that panel in multiple languages for the translations in my projects.

Reply:
I see your point, but I think it is rather OK because the message is just for Bot owner and developer.
Besides, for consistency, the Japanese Language piece also left those strings untranslated.